xen/xenops_interface.ml

```diff
@@ -16,6 +16,7 @@
  *)
 
 open Sexplib.Std
+include Xenops_types.TopLevel
```

Comment:
I'm not a big fan of `include` because it introduces a lot of names unqualified into the current scope. I understand why it is here, though.